rts/motoko-rts/src/principal_id.rs

```diff
@@ -225,7 +229,7 @@ unsafe fn base32_to_principal<M: Memory>(mem: &mut M, b: Value) -> Value {
     let mut data = blob.payload_const();
 
     // Every group of 5 characters will yield 6 bytes (due to the hypen)
-    let r = alloc_blob(mem, Bytes(((n.as_u32() + 4) / 5) * 6));
+    let r = alloc_blob(mem, TAG_BLOB_T, Bytes(((n.as_u32() + 4) / 5) * 6));
```

Comment:
I am not sure: Would the result maybe be a `TAG_BLOB_P`? (Just trying to infer it by the function name.)

Reply:
Unfortunately, this also required adjust the tests in crc32.rs